config/routes.rb

```diff
@@ -26,6 +26,16 @@
     delete 'logga_ut' => 'devise/sessions#destroy', as: :destroy_user_session
   end
 
+  # tags and exlinks
+  resources :ex_links
+
+  namespace :admin do
+    get 'ex_links/del_unused_tags' => 'ex_links#del_unused_tags'
+    get 'ex_links/check_dead' => 'ex_links#check_dead'
+    get 'ex_links/check_expired' => 'ex_links#check_expired'
+    resources :ex_links
```

Comment:
You need to use `link_to(ExLink.human_attribute_name(:delete_unused_tags), admin_ex_link_del_unused_tags)`.